model-api/src/commonMain/kotlin/org/modelix/model/api/IChildLink.kt

```diff
@@ -30,6 +30,15 @@ interface IChildLink : ILink {
     companion object {
         fun fromName(name: String): IChildLink = ChildLinkFromName(name)
     }
+
+    /**
+     * Whether children with this role can be ordered.
```

Comment:
Shouldn't this be more in the direction of whether ordering is significant?

It is probably also good to mention potential consequences if this false. for instance that multiple requests to children of a role are not guaranteed to return elements in the same order.

Reply:
Done. I extended KDoc to include how this impacts requesting children subsequently.